build.gradle

```diff
@@ -16,24 +16,24 @@
 import org.gradle.api.internal.plugins.osgi.OsgiHelper
 
 buildscript {
-  ext.kotlinVersion = '1.2.51'
+  ext.kotlinVersion = '1.3.31'
```

Comment:
consider splitting the commit in 2 (see other comments):

gradle and src changes purely related to the benchmarks vs unrelated gradle changes

I don't think the following gradle changes make sense in the same commit:
 - bump of the kotlin version
 - bump of the shadow plugin (I don't even remember it was used btw)
 - removing of the `sourceJar` security belt
 - bumping of the unbroken-dome plugin version
 - removing rsivaram from developers and adding @bsideup 

maybe at least split these gradle changes into their own commit, for future reference?